client/src/js/view/index.js

```diff
@@ -22,6 +30,10 @@ const view = (state, actions) => {
     clearTimeout(ADMIN_TIMEOUT);
   }
 
+  const setStreamRefreshInterval = () => {
+    setInterval(() => actions.setMsiStreamRefreshPending(true) && actions.stream.close(), STREAM_REFRESH_TIME);
```

Comment:
Use a semicolon instead of &&
`setInterval(() => { actions.setMsiStreamRefreshPending(true); actions.stream.close; }, STREAM_REFRESH_TIME)`